Orange/widgets/model/owstack.py

```diff
@@ -33,22 +33,26 @@ def add_main_layout(self):
     @Inputs.learners
     def set_learner(self, index: int, learner: Learner):
         self.learners[index] = learner
+        self.invalidate()
 
     @Inputs.learners.insert
     def insert_learner(self, index, learner):
         self.learners.insert(index, learner)
+        self.invalidate()
 
     @Inputs.learners.remove
     def remove_learner(self, index):
         self.learners.pop(index)
+        self.invalidate()
 
     @Inputs.aggregate
     def set_aggregate(self, aggregate):
         self.aggregate = aggregate
+        self.invalidate()
 
-    def handleNewSignals(self):
-        super().handleNewSignals()
-        self.apply()
```

Comment:
This should have called `unconditional_apply`. Now the base class defines `handleNewSignals` and handles them properly (I suppose).